
## Output of the trial, it was duplicated and surely the outputted files was overwritten upon , 
## so you need to by pass the second file as it is included in the first


*** Running kallisto on Pre_PD01_SRR5088813_Pt1_AD101148-6

[quant] fragment length distribution will be estimated from the data
[index] k-mer length: 31
[index] number of targets: 202,897
[index] number of k-mers: 115,705,407
[index] number of equivalence classes: 834,139
[quant] running in paired-end mode
[quant] will process pair 1: /home/emam/project19/RNA_seq/data/PRJNA356761_fqData_2/test/Pre/PD01_SRR5088813_GSM2420259_Pt1_Pre_AD101148-6_Homo_sapiens_RNA-Seq_1.fastq.gz
                             /home/emam/project19/RNA_seq/data/PRJNA356761_fqData_2/test/Pre/PD01_SRR5088813_GSM2420259_Pt1_Pre_AD101148-6_Homo_sapiens_RNA-Seq_2.fastq.gz
[quant] finding pseudoalignments for the reads ... done
[quant] processed 51,976,942 reads, 45,357,824 reads pseudoaligned
[quant] estimated average fragment length: 167.788
[   em] quantifying the abundances ... done
[   em] the Expectation-Maximization algorithm ran for 1,360 rounds
[bstrp] running EM for the bootstrap: 3

*** Running kallisto on Pre_PD01_SRR5088813_Pt1_AD101148-6

[quant] fragment length distribution will be estimated from the data
[index] k-mer length: 31
[index] number of targets: 202,897
[index] number of k-mers: 115,705,407
[index] number of equivalence classes: 834,139
[quant] running in paired-end mode
[quant] will process pair 1: /home/emam/project19/RNA_seq/data/PRJNA356761_fqData_2/test/Pre/PD01_SRR5088813_GSM2420259_Pt1_Pre_AD101148-6_Homo_sapiens_RNA-Seq_1.fastq.gz
                             /home/emam/project19/RNA_seq/data/PRJNA356761_fqData_2/test/Pre/PD01_SRR5088813_GSM2420259_Pt1_Pre_AD101148-6_Homo_sapiens_RNA-Seq_2.fastq.gz
[quant] finding pseudoalignments for the reads ... done
[quant] processed 51,976,942 reads, 45,357,824 reads pseudoaligned
[quant] estimated average fragment length: 167.788
[   em] quantifying the abundances ... done
[   em] the Expectation-Maximization algorithm ran for 1,360 rounds
[bstrp] running EM for the bootstrap: 3

*** Running kallisto on On_SD20_SRR5088832_Pt89_AE070987-5

[quant] fragment length distribution will be estimated from the data
[index] k-mer length: 31
[index] number of targets: 202,897
[index] number of k-mers: 115,705,407
[index] number of equivalence classes: 834,139
[quant] running in paired-end mode
[quant] will process pair 1: /home/emam/project19/RNA_seq/data/PRJNA356761_fqData_2/test/On/SD20_SRR5088832_GSM2420278_Pt89_On_AE070987-5_Homo_sapiens_RNA-Seq_1.fastq.gz
                             /home/emam/project19/RNA_seq/data/PRJNA356761_fqData_2/test/On/SD20_SRR5088832_GSM2420278_Pt89_On_AE070987-5_Homo_sapiens_RNA-Seq_2.fastq.gz
[quant] finding pseudoalignments for the reads ... done
[quant] processed 51,548,752 reads, 46,604,566 reads pseudoaligned
[quant] estimated average fragment length: 148.475
[   em] quantifying the abundances ... done
[   em] the Expectation-Maximization algorithm ran for 1,474 rounds
[bstrp] running EM for the bootstrap: 3

*** Running kallisto on On_SD20_SRR5088832_Pt89_AE070987-5

[quant] fragment length distribution will be estimated from the data
[index] k-mer length: 31
[index] number of targets: 202,897
[index] number of k-mers: 115,705,407
[index] number of equivalence classes: 834,139
[quant] running in paired-end mode
[quant] will process pair 1: /home/emam/project19/RNA_seq/data/PRJNA356761_fqData_2/test/On/SD20_SRR5088832_GSM2420278_Pt89_On_AE070987-5_Homo_sapiens_RNA-Seq_1.fastq.gz
                             /home/emam/project19/RNA_seq/data/PRJNA356761_fqData_2/test/On/SD20_SRR5088832_GSM2420278_Pt89_On_AE070987-5_Homo_sapiens_RNA-Seq_2.fastq.gz
[quant] finding pseudoalignments for the reads ... done
[quant] processed 51,548,752 reads, 46,604,566 reads pseudoaligned
[quant] estimated average fragment length: 148.475
[   em] quantifying the abundances ... done
[   em] the Expectation-Maximization algorithm ran for 1,474 rounds
[bstrp] running EM for the bootstrap: 3

*** Running kallisto on On_Unk01_SRR5088817_Pt23_AD486568-8

[quant] fragment length distribution will be estimated from the data
[index] k-mer length: 31
[index] number of targets: 202,897
[index] number of k-mers: 115,705,407
[index] number of equivalence classes: 834,139
[quant] running in paired-end mode
[quant] will process pair 1: /home/emam/project19/RNA_seq/data/PRJNA356761_fqData_2/test/On/Unk01_SRR5088817_GSM2420263_Pt23_On_AD486568-8_Homo_sapiens_RNA-Seq_1.fastq.gz
                             /home/emam/project19/RNA_seq/data/PRJNA356761_fqData_2/test/On/Unk01_SRR5088817_GSM2420263_Pt23_On_AD486568-8_Homo_sapiens_RNA-Seq_2.fastq.gz
[quant] finding pseudoalignments for the reads ... done
[quant] processed 49,225,898 reads, 44,111,534 reads pseudoaligned
[quant] estimated average fragment length: 157.726
[   em] quantifying the abundances ... done
[   em] the Expectation-Maximization algorithm ran for 1,495 rounds
[bstrp] running EM for the bootstrap: 3

*** Running kallisto on On_Unk01_SRR5088817_Pt23_AD486568-8

[quant] fragment length distribution will be estimated from the data
[index] k-mer length: 31
[index] number of targets: 202,897
[index] number of k-mers: 115,705,407
[index] number of equivalence classes: 834,139
[quant] running in paired-end mode
[quant] will process pair 1: /home/emam/project19/RNA_seq/data/PRJNA356761_fqData_2/test/On/Unk01_SRR5088817_GSM2420263_Pt23_On_AD486568-8_Homo_sapiens_RNA-Seq_1.fastq.gz
                             /home/emam/project19/RNA_seq/data/PRJNA356761_fqData_2/test/On/Unk01_SRR5088817_GSM2420263_Pt23_On_AD486568-8_Homo_sapiens_RNA-Seq_2.fastq.gz
[quant] finding pseudoalignments for the reads ... done
[quant] processed 49,225,898 reads, 44,111,534 reads pseudoaligned
[quant] estimated average fragment length: 157.726
[   em] quantifying the abundances ... done
[   em] the Expectation-Maximization algorithm ran for 1,495 rounds
[bstrp] running EM for the bootstrap: 3